BookingsApi/BookingsApi.IntegrationTests/Database/Commands/DeleteJusticeUserCommandTests.cs

```diff
@@ -92,7 +83,7 @@ private async Task<JusticeUser> SeedJusticeUser(string username, Guid allocatedH
             {
                 ContactEmail = username,
                 Username = username,
-                UserRoleId = (int)UserRoleId.Vho,
+                //UserRoleIds = new []{(int)UserRoleId.Vho},
```

Comment:
as per a previous comment, it might be worth assigning the user roles here to put the user into a realistic/valid state